src/meshcat/visualizer.py

```diff
@@ -68,11 +68,11 @@ def get_scene(self):
         # we receive the HTML as utf-8-encoded, so decode here
         return self.zmq_socket.recv().decode('utf-8')
 
-    def get_image(self, save_path=""):
-        cmd_data = CaptureImage(save_path).lower()
+    def get_image(self):
+        cmd_data = CaptureImage().lower()
         self.zmq_socket.send_multipart([
             cmd_data["type"].encode("utf-8"),
-            cmd_data["save_path"].encode("utf-8"),
+            "".encode("utf-8"),
```

Comment:
This is what I was talking about. This makes sure `frames` in `ZMQSocketBridge.forward_to_websockets` has the right length.

Ideally we could change that method to be more flexible, too

Reply:
Ok, makes sense. I think this is fine for now.